.pre-commit-config.yaml

```diff
@@ -10,13 +10,12 @@ repos:
       - id: requirements-txt-fixer
       - id: no-commit-to-branch
         args: ['--branch', 'main']
-  - repo: local
+
+  - repo: https://github.com/psf/black
```

Comment:
Question... Why not include pylint here instead of github actions? Example of how we did in convert2rhel: https://github.com/oamg/convert2rhel/blob/main/.pre-commit-config.yaml#L34-L45

Reply:
I can, but it was kinda confusing.. should I run pylint in pre-commit with newest version (because that is possible) and if so should it be only in pre-commit and not available via make at all? because to run in in virtualenv with python2.7 I would have to keep the version lower

But I guess why not :shrug: black is also running in pre-commit when technically we wouldn't be able to run it in virtualenv because it need python3.6+

Reply:
If we put it into pre-commit, we don't need to install/call that in a virtualenv. pre-commit will do that job for us every time we want to validate something. 

Also, I will ask someone to enable pre-commit.ci for this repository, so those hooks will run in every pr/commit.

Reply:
Alright.. my question was aimed to the fact that if I would want to keep running it locally via make command inside virtualenv - just to have that option to test it manually same way as pytest, then I would have to maintain the pylint version in requirements 

But let's keep it only in pre-commit, why not, that can be run manually as well

Reply:
alright, done .. :)